equipment-models/src/main/java/com/telecominfraproject/wlan/equipment/models/ElementRadioConfiguration.java

```diff
@@ -48,6 +50,46 @@
 
     private Boolean deauthAttackDetection;
     private Set<ChannelPowerLevel> allowedChannelsPowerLevels = new HashSet<>();
+    
+    private static final Map<RadioType, Integer> defaultChannelNumberMap = new EnumMap<>(RadioType.class);
+    private static final Map<RadioType, Integer> defaultBackupChannelNumberMap = new EnumMap<>(RadioType.class);
+    
+    public final static int DEFAULT_CHANNEL_NUMBER_2DOT4GHZ = 6;
+    public final static int DEFAULT_BACKUP_CHANNEL_NUMBER_2DOT4GHZ = 11;
+    public final static int DEFAULT_CHANNEL_NUMBER_5GHZ = 36;
+    public final static int DEFAULT_BACKUP_CHANNEL_NUMBER_5GHZ = 44;
+    public final static int DEFAULT_CHANNEL_NUMBER_5GHZL = 36;
+    public final static int DEFAULT_BACKUP_CHANNEL_NUMBER_5GHZL = 44;
+    public final static int DEFAULT_CHANNEL_NUMBER_5GHZU = 149;
+    public final static int DEFAULT_BACKUP_CHANNEL_NUMBER_5GHZU = 157;
+    
+    static {
+        defaultChannelNumberMap.put(RadioType.is2dot4GHz, DEFAULT_CHANNEL_NUMBER_2DOT4GHZ);
+        defaultChannelNumberMap.put(RadioType.is5GHz, DEFAULT_CHANNEL_NUMBER_5GHZ);
+        defaultChannelNumberMap.put(RadioType.is5GHzL, DEFAULT_CHANNEL_NUMBER_5GHZL);
+        defaultChannelNumberMap.put(RadioType.is5GHzU, DEFAULT_CHANNEL_NUMBER_5GHZU);
+        
+        defaultBackupChannelNumberMap.put(RadioType.is2dot4GHz, DEFAULT_BACKUP_CHANNEL_NUMBER_2DOT4GHZ);
+        defaultBackupChannelNumberMap.put(RadioType.is5GHz, DEFAULT_BACKUP_CHANNEL_NUMBER_5GHZ);
+        defaultBackupChannelNumberMap.put(RadioType.is5GHzL, DEFAULT_BACKUP_CHANNEL_NUMBER_5GHZL);
+        defaultBackupChannelNumberMap.put(RadioType.is5GHzU, DEFAULT_BACKUP_CHANNEL_NUMBER_5GHZU);
+    }
+    
+    public static Integer getDefaultChannelNumber(RadioType radioType) {
+        Integer channelNumber = defaultChannelNumberMap.get(radioType);
+        if (channelNumber == null) {
+            return DEFAULT_CHANNEL_NUMBER_2DOT4GHZ;
+        }
+        return channelNumber;
+    }
+    
+    public static Integer getDefaultBackupChannelNumber(RadioType radioType) {
+        Integer backupChannelNumber =  defaultBackupChannelNumberMap.get(radioType);
+        if (backupChannelNumber == null) {
+            return DEFAULT_BACKUP_CHANNEL_NUMBER_2DOT4GHZ;
```

Comment:
Same comment as previous, why the 2.4GHz channel if the radio type is not found in the default channel map